test-suite/js/src/test/scala/org/scalajs/testsuite/library/WrappedSetTest.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Assume.assumeTrue
+import org.junit.{ BeforeClass, Test }
+import org.scalajs.testsuite.utils.Platform._
+
+import scala.collection.mutable
+import scala.scalajs.js
+
+object WrappedSetTest {
+  @BeforeClass
+  def assumeRuntimeSupportsSet(): Unit = {
+    assumeTrue("Assume Symbol and Set exists in Global",
+      jsSymbols && js.typeOf(js.constructorOf[js.Set[_]]) != "undefined")
```

Comment:
```suggestion
        jsSymbols && js.typeOf(js.constructorOf[js.Set[_]]) != "undefined")
```
Here also: why test Symbols?